plugins/builds/artifacts/unzip.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = schema.models.build.base.extract('id');
+
+module.exports = config => ({
+    method: 'POST',
+    path: '/builds/{id}/artifacts/unzip',
+    options: {
+        description: 'Extract a ZIP for build artifacts',
+        notes: 'Extract a specific ZIP for build artifacts',
+        tags: ['api', 'builds', 'artifacts'],
+        auth: {
+            strategies: ['token'],
+            scope: ['build']
```

Comment:
Due to some reason, unzip would be failed and user have to rerun the build.
Do we need to consider this endpoint should include `admin` scope for a case of failure?

Reply:
It's good idea to add admin scope for a case of failure.
We'll create another PR for that after fixing the design doc.